improver_tests/cli/test_init.py

```diff
@@ -181,7 +181,18 @@ def test_with_output_no_compression(self, m):
         # pylint disable is needed as it can't see the wrappers output kwarg.
         # pylint: disable=E1123
         result = wrapped_with_output(2, output="foo", compression_level=0)
-        m.assert_called_with(4, "foo", 0)
+        m.assert_called_with(4, "foo", 0, None)
+        self.assertEqual(result, None)
+
+    @patch("improver.utilities.save.save_netcdf")
+    def test_with_output_with_least_significant_figure(self, m):
+        """Tests that save_netcdf is called with object and string and compression_level=0 """
```

Comment:
Docstrings for with_output tests need updating to add `least_significant_figure`

Reply:
Woops! Thanks for this :-)